UPGRADE-6.7.md

```diff
@@ -1,20 +1,86 @@
+This ensures that our administration stays future-proof and we can make use of the most recent Vue 3 features.
+
+**What does that mean, can we link to a guide or something?**
+
+**Note:** This change can be activated separately with the `DISABLE_VUE_COMPAT` feature flag.
+
+## Migration from Vuex to Pinia
+For Vue 3 the default state management library has become Pinia, therefore we are migrating from Vuex to Pinia. to stay as close to the default as possible.
+When you use default stores in your plugin you need to switch from `Shopware.State` (Vuex) to `Shopware.Store` (Pinia).
+Adding your own Vuex stores is still possible, however it is recommended that you switch to Pinia as well.
+
+**TBD, can we have a realistic example how this looked like before and what actually needs to change?
+
+For more information refer to the [docs](https://developer.shopware.com/docs/resources/references/adr/2024-06-17-replace-vuex-with-pinia.html#replace-vuex-with-pinia).
+
+# Cache Rework
+**Note:** Those changes can be activated separately with the `cache_rework` feature flag.
+
+## Delayed Cache Invalidation
+The cache invalidation will be delayed by default. This means that the cache will be invalidated in regular intervals and not immediately.
+This will lead to better cache hit rates and way less (duplicated) cache invalidations, which will improve efficiency and scalability of the system.
+As this feature is now active by default the previous `shopware.cache.invalidation.delay` configuration is removed.
+
+The default interval is 5 min, this can be changed by adjusting the run interval of the `shopware.invalidate_cache` scheduled task.
+
+If you sent an API request with critical information, where the cache should be invalidated immediately, you can set the `sw-force-cache-invalidate` header on your request.
+```
+POST /api/product
+sw-force-cache-invalidate: 1
+```
+
+To manually clear all the stale caches you can either run the `cache:clear:delayed` command or use the `/api/_action/cache-delayed` API endpoint.
+```
+bin/console cache:clear:delayed
+```
+```
+DELETE /api/_action/cache-delayed
+```
+
+For debugging there is the `cache:watch:delayed` command available, to watch the cache tags that are stored in the delayed cache invalidation queue.
+```
+bin/console cache:watch:delayed
+```
+
+## Removal of Store-API route caching
+The Store-API route caching has been removed. This means that the `Cached*Route` classes will be removed.
+This solves some weird states when the HTTP-Cache was invalidated separately from the route cache.
+Additionally, the cache hit rate for the Store-API was low, so the performance impact should be minimal, but the amount of cache items and cache invalidations will be reduced.
+This overall should lead to more effective cache resource usage.
+
+## Use ESI includes for Header and Footer
```

Comment:
@mitelg can you add ESI related changes here as soon as we know what we need to break?